lib/home_page.dart

```diff
@@ -1,9 +1,12 @@
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_dojo/repositories_page.dart';
 
 class HomePage extends StatefulWidget {
   HomePage({Key key}) : super(key: key);
 
+  static const routeName = '/';
```

Comment:
If you need to open this page for some reason at some time, the navigation will not work with this slash (/). You should specific a descriptive route name for this page.

Place this slash just only inside the lib/main.dart file.